functions/webhooks/twitter/TwitterToFlex.ts

```diff
@@ -74,12 +75,43 @@ const isValidTwitterPayload = (event: Body, consumerSecret: string): boolean =>
 /**
  * Retrieves a channel by sid or uniqueName
  */
-const retrieveChannel = (context: Context<EnvVars>, uniqueName: string) =>
-  context
+const retrieveChannel = async (
+  context: Context<EnvVars>,
+  uniqueSenderName: string,
+): Promise<string | undefined> => {
```

Comment:
I believe it's now retrieving a channelSid, instead of a channel, right? if that's the case, I suggest changing the function name to be `retrieveChannelSid` and update the function comment above.